apps/site-nft/src/components/MainNavigation/MainNavigation.stories.tsx

```diff
@@ -6,19 +6,16 @@ import { withAccounts, withLanguageProvider, withMediaContext } from '@thecointe
 import styles from '../App/styles.module.less';
 
 export default {
-  title: 'NFT/Header',
+  title: 'NFT/Navigation',
```

Comment:
We have several 'Header' classes, so renamed this one to 'Navigation' to make it easier to find/thinkabout